modules/loader/src/main/java/com/opengamma/strata/loader/CharSources.java

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright (C) 2017 - present by OpenGamma Inc. and the OpenGamma group of companies
+ *
+ * Please see distribution for license.
+ */
+package com.opengamma.strata.loader;
+
+import java.io.File;
+import java.nio.charset.Charset;
+import java.nio.file.Path;
+
+import com.google.common.base.Charsets;
+import com.google.common.io.ByteSource;
+import com.google.common.io.CharSource;
+import com.google.common.io.Files;
+import com.opengamma.strata.collect.io.ResourceLocator;
+
+/**
+ * Helper that allows {@link CharSource} objects to be created.
+ */
+public final class CharSources {
+
+  /**
+   * Obtains an instance of {@link CharSource} from a file name, specified as a String.
+   *
+   * @param fileName the file name, as a String
```

Comment:
In Strata, we have a very specific two spaces rule between the parameter name and description. It only tends to get used in our open source projects now